app/views/generic_object_definition/show.html.haml

```diff
@@ -3,3 +3,9 @@
     = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
   - else
     = render :partial => 'layouts/textual_groups_generic'
+
+  %generic-object-definition-toolbar#generic_object_definition_show_form{'generic-object-definition-id' => @record.id,
+                                                                         'redirect-url'                 => "/#{controller_name}/show_list",}
+:javascript
+  miq_bootstrap('#generic_object_definition_show_form');
```

Comment:
And here.. I'll say fine for this PR :)...

So just FYI Aparna..

we're trying to remove all those specific things from these `show` partials, so that `miq_bootstrap` will have to go as well..
but the idea is that instead, the controllers would have a method which gets called from all the show partials, and can create that `miq_bootstrap` in a declarative way, maybe ideally with the option to create that component element as well.

But that's really just an idea for now, without any specifics worked out.

Reply:
It's in the TODO list that is growing...